va/va_dec_av1.h

```diff
@@ -444,7 +444,8 @@ typedef struct  _VADecPictureParameterBufferAV1
     } pic_info_fields;
 
     /** \brief Supper resolution scale denominator.
-     *  value range [9..16]
+     *  When use_superres=1, superres_scale_denominator must be in the range [9..16].
+     *  When use_superres=0, superres_scale_denominator must be 8.
```

Comment:
when use_superres==0 should has  same effect with superres_scale_denominator == 8.
they should have same meaning.  if (use_superres == 0 || supperres_scale_denominator == 8) supper res is disabled, right?

Reply:
Whenever, use_superres = 0, supperres_scale_denominator will be set to 8 Carl. From driver code - cmd.DW9.SuperResDenom = m_picParams->PicFlags.fields.use_superres ? params->m_picParams->superres_scale_denominator : 8;

Reply:
I mean, there are already use_supperes == 0 which is similiar with superres_scale_denominator ==8. [8..16] & [9..16] looks same. application could get expected result from the combination of these two varaible , only difference is [8..16] is aligned with spec, and it is straightforward, right?

Reply:
Our intent is to make sure that we show Super resolution scale denominator value range is [8..16] as per AV1 spec. However, the extra details provide information to the reader on when we use 8 vs [9..16].